README.md

```diff
@@ -51,23 +51,22 @@ On the roadmap:
 
 ## Running the Event Management Agent
 
-### Prerequisites:
-
-* Java 11 (AdoptOpenJDK 11.0.14+ https://adoptium.net/temurin/releases)
-* Maven
-* Docker (for running the Event Management Agent with MySQL database)
-* Event Management Agent Region (for cloud mode)
-
 ### Minimum hardware requirements
 
 The Event Management Agent was tested to run with
 
 * 1 CPU
 * 1 GB RAM
 
-### Spring-boot properties
+### Prerequisites:
 
-Specific properties are required to run this spring boot application in cloud mode. </br>
+* Java 11 (AdoptOpenJDK 11.0.14+ https://adoptium.net/temurin/releases)
+* Maven
+* Docker (recommended)
+
+### Event Management Agent connection file
+
+Specific properties are required to run the Event Management Agent in the connected to Event Portal mode. </br>
 These properties are defined in the connection file that can be downloaded from the Event Portal's Event Management Agent definition. </br>
 The steps for generating the connection file are listed over here : https://docs.solace.com/Cloud/Event-Portal/event-portal-collect-runtime-data.htm#creating_connection_file </br>
 Replace the application.yml present in the location : event-management-agent/service/application/src/main/resources with the EMA connection file. </br>
```

Comment:
I think that we need to do away with the application.yml compiled into the java jar. It's bad form to compile identifiers w/ security implications into an exec. Executed from docker, this file is passed to executable as a spring boot over-ride. We should recommend the same method when running the agent as a process / command line.

Reply:
I agree. We need to plan this work.